entrypoints/docker.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+set -e
+
+if [ -f tmp/pids/server.pid ]; then
+    rm tmp/pids/server.pid
+fi
+
+bin/rails db:create && \
```

Comment:
Please copy over the code to create development_env.rb if it doesn't already exist. https://github.com/King-County-Equity-Now/decidim-seattle/blob/main/script/dev-setup#L62